packages/slice-machine/rollup-server.config.js

```diff
@@ -15,7 +15,7 @@ export default {
         "process.env.NEXT_PUBLIC_SM_UI_SEGMENT_KEY": process.env
```

Comment:
I'd remove the `NEXT` from it's name, it was used a convention for nextjs to pick it up i think.

Reply:
We could but we would also have to make an new env variable, some where.... I'm not sure where though :)